package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@videojs/svg-sprite-generator",
+  "version": "1.0.0",
+  "description": "Effortlessly customize Video.js player icons",
+  "license": "MIT",
+  "author": {
+    "name": "amtins",
+    "email": "amtins.dev@gmail.com",
+    "url": "https://github.com/amtins"
+  },
+  "type": "module",
+  "bin": {
+    "vjs-svg-sprite": "./bin/cli.js"
+  },
+  "scripts": {
+    "build": "node index.js",
```

Comment:
I am trying to think if there are any other scripts we may want to include here.. Would we want to include `lint`, or maybe `clean` to remove the dist?